tests/integration/container/tests/utils/driver_helper.ts

```diff
@@ -117,8 +117,8 @@ export class DriverHelper {
       props["monitoring_query_timeout"] = 3000;
     } else if (engine === DatabaseEngine.MYSQL && performance) {
       props["connectTimeout"] = 3000;
-      props["monitoring_mysqlQueryTimeout"] = 3000;
-      props["mysqlQueryTimeout"] = 120000;
+      props["monitoring_wrapperQueryTimeout"] = 3000;
```

Comment:
It might be too late (or might be not) to consolidate our parameter names. They should use either camel syntax or underscores. It seems we have a mix of both. Probably it's out of scope of this PR.